pkg/trait/container.go

```diff
@@ -74,12 +74,12 @@ type containerTrait struct {
 	ServicePort int `property:"service-port" json:"servicePort,omitempty"`
 	// To configure under which service port name the container port is to be exposed (default `http`).
 	ServicePortName string `property:"service-port-name" json:"servicePortName,omitempty"`
-
 	// The main container name. It's named `integration` by default.
 	Name string `property:"name" json:"name,omitempty"`
 	// The main container image
 	Image string `property:"image" json:"image,omitempty"`
-
+	// The pull policy: Always|Never|IfNotPresent
+	ImagePullPolicy string `property:"image-pull-policy" json:"imagePullPolicy,omitempty"`
```

Comment:
I wonder if the `corev1.PullPolicy` type could be used directly here?

Reply:
Yes, it's cleaner this way. I've committed the changes, please have a further look.